toolchain/check/testdata/interop/cpp/no_prelude/function.carbon

```diff
@@ -4,124 +4,144 @@
 //
 // AUTOUPDATE
 // TIP: To test this file alone, run:
-// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/interop/cpp/no_prelude/function_decl.carbon
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/interop/cpp/no_prelude/function.carbon
 // TIP: To dump output, run:
-// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/interop/cpp/no_prelude/function_decl.carbon
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/interop/cpp/no_prelude/function.carbon
 
-// --- function_decl.h
+// ============================================================================
+// global
+// ============================================================================
+
+// --- global.h
 
 void foo();
 
-// --- import_function_decl.carbon
+// --- import_global.carbon
 
 library "[[@TEST_NAME]]";
 
-import Cpp library "function_decl.h";
+import Cpp library "global.h";
 
 fn MyF() {
   Cpp.foo();
 }
 
-// --- fail_import_function_decl_use_different_name.carbon
+// --- fail_import_global_use_different_name.carbon
 
 library "[[@TEST_NAME]]";
 
-import Cpp library "function_decl.h";
+import Cpp library "global.h";
 
 fn MyF() {
-  // CHECK:STDERR: fail_import_function_decl_use_different_name.carbon:[[@LINE+4]]:3: error: member name `bar` not found in `Cpp` [MemberNameNotFoundInInstScope]
+  // CHECK:STDERR: fail_import_global_use_different_name.carbon:[[@LINE+4]]:3: error: member name `bar` not found in `Cpp` [MemberNameNotFoundInInstScope]
   // CHECK:STDERR:   Cpp.bar();
   // CHECK:STDERR:   ^~~~~~~
   // CHECK:STDERR:
   Cpp.bar();
 }
 
-// --- function_special_name_decl.h
+// ============================================================================
+// special_name
```

Comment:
No need to rename the files to match here.
```suggestion
// Carbon keyword name
```

Reply:
Done.